app/routes/overview/components/PublicFrontpage.tsx

```diff
@@ -116,59 +119,71 @@ const LatestArticle = ({ frontpage }: Pick<Props, 'frontpage'>) => (
+    buttonText: 'Sjekk ut studiet',
+    link: 'https://www.ntnu.no/studier/mtdt',
+  },
+  {
+    title: 'Kommunikasjonsteknologi og digital sikkerhet',
+    image: komtekGraphic,
+    description:
+      'Vi bruker stadig mer av livene våre på nett, på jobb som i fritid. Kommunikasjonsteknologi og digital sikkerhet blir stadig viktigere i en digital verden.',
+    buttonText: 'Sjekk ut studiet',
+    link: 'https://www.ntnu.no/studier/mtkom',
+  },
+  {
+    title: 'For bedrifter',
+    image: forCompaniesGraphic,
+    description:
+      "Her finner du som bedriftsrepresentant informasjon om Abakus' prosedyrer for bedriftspresentasjoner, og andre nyttige fakta.",
+    buttonText: 'Undersøk muligheter',
+    link: '/pages/bedrifter/for-bedrifter',
+  },
+  {
+    title: 'readme',
+    image: readmeGraphic,
+    description:
+      'Abakus har sitt eget magasin, readme. Her kan du lese om hva vi driver med, og få et innblikk i oss som organisasjon.',
+    buttonText: 'Utforsk magasiner',
+    link: 'https://readme.abakus.no/',
+  },
+];
+
 const UsefulLinks = () => (
   <>
     <h2 className="u-mb">Nyttige lenker</h2>
-    <ul>
-      <li>
-        <Link to="/articles/414">
-          <i className="fa fa-caret-right" /> Fadderperioden 2022
-        </Link>
-        <div className={styles.linkDescription}>
-          Informasjon om fadderperioden 2022
-        </div>
-      </li>
-      <li>
-        <a href="https://www.ntnu.no/studier/mtdt" target="blank">
-          <i className="fa fa-caret-right" /> Datateknologi
-        </a>
-        <div className={styles.linkDescription}>
-          Datateknologi er en helt sentral del av alle fremtidsrettede
-          teknologier, som for eksempel kunstig intelligens, medisinsk teknologi
-          og søkemotorteknologi.
-        </div>
-      </li>
-      <li>
-        <a href="http://www.ntnu.no/studier/mtkom" target="blank">
-          <i className="fa fa-caret-right" /> Kommunikasjonsteknologi og digital
-          sikkerhet
-        </a>
-        <div className={styles.linkDescription}>
-          Vi bruker stadig mer av livene våre på nett, på jobb som i fritid.
-          Kommunikasjonsteknologi og digital sikkerhet blir stadig viktigere i
-          en digital verden.
-        </div>
-      </li>
-      <li>
-        <Link to="/pages/bedrifter/for-bedrifter">
-          <i className="fa fa-caret-right" /> For bedrifter
-        </Link>
-        <div className={styles.linkDescription}>
-          Her finner du som bedriftsrepresentant informasjon om Abakus{"' "}
-          prosedyrer for bedriftspresentasjoner og andre nyttige fakta.
-        </div>
-      </li>
-      <li>
-        <a href="https://readme.abakus.no">
-          <i className="fa fa-caret-right" /> {readmeIfy('readme')}
+
+    <Flex wrap justifyContent="center" gap={40}>
+      {usefulLinksConf.map((item) => (
+        <a href={item.link} key={item.title} rel="noreferrer" target="_blank">
```

Comment:
Hmmm, not sure about this. I think it's better to use `target=_blank` only for external sites, and keep the internal ones as normal links.

Reply:
Sure. I couldn't use `<Link />` since, as you can see, there are both internal and external links. I suppose I can add a `isInternalLink` field to the conf.